### PR TITLE
Lifeweb poster removal

### PR DESCRIPTION
**Pull request by Al-1ce**
## About The Pull Request
There's some goober out there who thought copying Lifeweb lore and putting it here would be a great idea. I physically cringed upon seeing it and thus this PR was born.

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
del: Removed out of place reference to the Cult of Cons, from Lifeweb. 
/:cl:

### CODE REVIEW

**Comment by ErwinSnowden** [comment]
might want to get rid of these since they reference the thanati, dreamer and other completely non-canon to fallout shit. whoever thought they were cool by adding this shit needs to blow their brains out with a thirty eight magnum. intel suggests it is the fortune13 contributor known as "john-dang". please point and laugh at this user
image
image
image
image

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -566,24 +566,19 @@
...
-
+	
 /obj/structure/sign/poster/prewar/poster72
 	name = "old military poster"
 	desc = "This picture glorifies some military commander of the past. You can't tell if it was before or after the collapse."
 	icon_state = "poster72"
-
-/obj/structure/sign/poster/prewar/poster73
+	
+	/obj/structure/sign/poster/prewar/poster73
```
Comment (RonBag) [inline code/game/objects/effects/contraband.dm:579]:
```suggestion
/obj/structure/sign/poster/prewar/poster73
```

Comment (Al-1ce) [reply code/game/objects/effects/contraband.dm]: Done.

**Comment by Al-1ce** [comment]
Closed because every F13 server seems to be ran by retarded discord no-lifers who can't live a moment without drama.

/salut

**Comment by Al-1ce** [comment]
I had some hope in this server. It had everything most other F13s lacked; an active repo, decent-ish community, solid developers..

BUUT some retarded mouth breather fresh out the womb & frothing at the mouth HAS to come along and axe everything because his discord kitten unfriended him. Or whatever stupid bullshit happened. It's all the same retarded discord drama in the end.

The ride never ends.
o7

**Pull request closed without merging**